server/routes/v1/index.js

```diff
@@ -16,12 +16,12 @@ router.get('/entries/:id', (req, res) => {
 });
 
 router.post('/entries', (req, res) => {
-  if (!req.body.post || req.body.post.length < 2) {
-    res.status(400).send({ status: 400, message: 'Oops - Bad Request' });
+  if (!req.body) {
```

Comment:
you should check for the values that you need in the request body such as title and content rather than the presence of a body in the request object.